index.js

```diff
@@ -1,7 +1,9 @@
-const objectToString = Object.prototype.toString;
-
 export function isUint8Array(value) {
-	return value && objectToString.call(value) === '[object Uint8Array]';
+	if (!value) {
+		return false;
+	}
+
+	return value.constructor === Uint8Array;
 }
```

Comment:
I'm not sure if this is a safe change. I use `objectToString.call(value) === '[object Uint8Array]'` so it will work across realms. You could maybe use both, just to be sure. This could be fast initial check.